oauth2_provider/migrations/0005_auto_20211222_2352.py

```diff
@@ -0,0 +1,39 @@
+import django.db.models.deletion
+from django.conf import settings
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
+        ('oauth2_provider', '0004_auto_20200902_2022'),
+    ]
+
+    operations = [
```

Comment:
**question**: Did we figure out what changes these operations are doing?

I don't see any changes to the models file that would cause this. Maybe its just a Django 4 related change?

Reply:
Yeah I couldn't figure it out exactly why they needed this noop PR but I am pretty sure it is due to django 4
https://github.com/jazzband/django-oauth-toolkit/pull/1056